django/contrib/postgres/fields/jsonb.py

```diff
@@ -0,0 +1,103 @@
+import json
+
+from psycopg2.extras import Json
+
+from django.contrib.postgres import forms, lookups
+from django.core import exceptions
+from django.db.models import Field, Transform
+from django.utils.translation import ugettext_lazy as _
+
+__all__ = ['JSONField']
+
+
+class JSONField(Field):
+    empty_strings_allowed = False
+    description = _('A JSON object')
+    default_error_messages = {
+        'invalid': _("'%(value)s' value must be valid JSON."),
```

Comment:
A wonder if repeating the value in the message is necessary considering that it's present in the invalid form. In the event of a large JSON blob, the message could be unslightly.